packages/api/src/external/commonwell/patient.ts

```diff
@@ -272,157 +327,109 @@ export async function update(
   });
 
   if (cwUpdateEnabled) {
-    await setPatientDiscoveryStatus({
-      patientId: patient.id,
-      cxId: patient.cxId,
+    const discoveryRequestId = requestId ?? uuidv7();
+    const discoveryStartedAt = new Date();
+    const augmentedPatient = augmentDemographics
+      ? await augmentPatientDemograhpics(patient)
+      : patient;
+    await updatePatientDiscoveryStatus({
+      patient,
       status: "processing",
+      discoveryRequestId,
+      discoveryFacilityId: facilityId,
+      discoveryStartedAt,
+      rerunPdOnNewDemographics,
+      augmentedDemographics: augmentDemographics
```

Comment:
Why are we storing the `augmentedDemographics` on the patient source? I dont see it being used anywhere else

Reply:
Auditing purposes -- mentioned it in the design doc but we never reviewed it. Happy to remove it if we feel this paper trail isn't necessary.

Reply:
I see - this is fine for now then should help with debugging

Reply:
1. When this is executed a second time (another patient update), `augmentedDemographics` could be removed from the patient, no?
2. Can't we store this on `cw_patient_data`?

Reply:
Irrelevant now: removed.